prime-dotnet-webapi/Controllers/HealthAuthoritiesController.cs

```diff
@@ -240,5 +246,56 @@ public async Task<ActionResult> UpdatePharmanetAdministratorContacts(int healthA
             await _healthAuthorityService.UpdateContactsAsync<HealthAuthorityPharmanetAdministrator>(healthAuthorityId, contacts);
             return NoContent();
         }
+
+        // GET: api/health-authorities/5/vendors/5/sites
+        /// <summary>
+        /// Checks if the specified vendor is in use by a Health Authority site
```

Comment:
Could probably simplify this to
```returns a list of site ids of a given vendor and health authority```
Since as written it is a little miss leading.

Reply:
Simplified